brokerapi/brokers/brokers_test.go

```diff
@@ -193,6 +193,13 @@ var _ = Describe("Brokers", func() {
 				}
 			}
 		})
+
+		It("should error if plan ids are not supplied", func() {
+			os.Setenv("PRECONFIGURED_PLANS", fakes.PlanNoId)
+			_, err := brokers.New(logger)
+			Expect(err).To(HaveOccurred())
+			os.Setenv("PRECONFIGURED_PLANS", fakes.PreconfiguredPlans)
```

Comment:
it's in a before each, I put that there while I was debugging and forgot to take it out. Thanks! Good catch!